### PR TITLE
Initial implementation of Utilities

### PR DESCRIPTION
**Pull request by zspencer**
See: https://github.com/zinc-collective/convene/issues/252

### CODE REVIEW

File: convene-web/app/lib/furniture.rb [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
     videobridge_by_jitsi: Furniture::VideobridgeByJitsi,
     markdown_text_block: Furniture::MarkdownTextBlock
-  }
+  }.freeze
 
-  # Allows Furniture to expose their controllers
+  # Appends each {Furniture}'s CRUD actions
   def self.append_routes(routing_context)
-    REGISTRY.values.each do |furniture|
+    REGISTRY.each_value do |furniture|
```
Comment (user512) [inline convene-web/app/lib/furniture.rb:14]: TIL about `each_value`

**Review by user512 — APPROVED** [review]
I feel like I am going to need to have you to demo this a bit to completely understand what's going on but this looks good to me.

File: convene-web/app/models/space.rb [hunk 2]
```diff
@@ -56,4 +53,22 @@ def unlocked?
...
+  has_many :utility_hookups
+
+  def jitsi_meet_domain
+    jitsi_hookup = utility_hookups.find_by(utility_slug: :jitsi)
+    return if jitsi_hookup.blank?
+
+    Utilities.from_utility_hookup(jitsi_hookup).meet_domain
+  end
+
+  def hookups
```
Comment (anaulin) [inline convene-web/app/models/space.rb:69]: Is this used anywhere? I couldn't find a usage example in this PR (but its big, so likely I missed it?). Looking at the implementation, it reads like it returns Utilities, not UtilityHookups, so I was trying to understand how it is used.

**Review by anaulin — APPROVED** [review]
It's starting to feel quite fleshed out! 🍖

File: convene-web/app/models/space.rb (hunk 2 above)
Comment (zspencer) [reply convene-web/app/models/space.rb:69]: Probably not! Thank you!

**Pull request merged**